subsys/usb/device/class/cdc_acm.c

```diff
@@ -527,8 +527,7 @@ static int cdc_acm_fifo_fill(const struct device *dev,
 		dev_data, len, ring_buf_space_get(dev_data->tx_ringbuf));
 
 	if (!dev_data->configured || dev_data->suspended) {
-		LOG_WRN("Device not configured or suspended, drop %d bytes",
```

Comment:
See https://github.com/zephyrproject-rtos/zephyr/pull/76171#discussion_r1687022631 for a discussion on why checking for suspended here might not be the best idea (it leads to autosuspend issues if device support Remote Wakeup).